payments-core/src/test/java/com/stripe/android/model/parsers/ElementsSessionJsonParserTest.kt

```diff
@@ -309,7 +317,7 @@ class ElementsSessionJsonParserTest {
                 countryCode = "CA",
                 created = 1,
                 isLiveMode = false,
-                usage = null,
+                usage = StripeIntent.Usage.OffSession,
```

Comment:
It is a bit surprising that this test passed before the recent changes. Can we verify that there isn't something hiding here?

Reply:
Discussed offline. Test setup was incorrect before this change, so that’s why the test passed with a `null` SFU value.